spec/features/journal_administration_spec.rb

```diff
@@ -67,7 +67,10 @@
         role = journal_page.add_role
         role.name = "whatever"
         role.save
-        expect(role).to have_name("whatever")
+        # NOTE: `expect(role).to have_name("whatever")` fails.
+        # Re-finding it works.
+        new_role = journal_page.find_role("whatever")
```

Comment:
I'm not sure about this issue myself, but I think it's fine to leave a comment and move on.